src/app/public/modules/ag-grid/cell-editors/cell-editor-datepicker/cell-editor-datepicker.component.scss

```diff
@@ -7,4 +7,10 @@ sky-datepicker {
     box-shadow: none;
     border-right: 1px solid $sky-border-color-neutral-medium;
   }
+
+  ::ng-deep {
+  .sky-input-group-datepicker-btn {
```

Comment:
Instead of using `ng-deep`, I'd suggest disabling view encapsulation. It's what we've done when theming other components.